taxonomies/Message.ttl

```diff
@@ -436,3 +436,76 @@ ids:LogMessage a owl:Class;
     rdfs:label "Log Message"@en ;
     rdfs:comment "Log Message which can be used to transfer logs e.g. to the clearing house."@en.
 
+
+# App-related Messages
+# ---------------------
+
+ids:AppRegistrationRequestMessage a owl:Class;
+    rdfs:subClassOf ids:RequestMessage ;
+    rdfs:label "App Registration Request Message"@en ;
+    rdfs:comment "Message that asks for registration of a new app to the appStore. Payload contains app data."@en ;
+.
+
+ids:AppRegistrationResponseMessage a owl:Class;
+    rdfs:subClassOf ids:ResponseMessage ;
+    rdfs:label "App Registration Response Message"@en ;
+    rdfs:comment "Message that follows up an AppRegistrationRequestMessage and contains the app registration confimation."@en ;
+.
+
+ids:AppUpdateRequestMessage a owl:Class;
+    rdfs:subClassOf ids:RequestMessage ;
+    rdfs:label "App Update Request Message"@en ;
+    rdfs:comment "Message that asks for Update of an existing app to the appStore. Payload contains app data."@en ;
+    idsm:validation [
+                        idsm:forProperty ids:affectedDataAppResource;
+                        idsm:constraint idsm:NotNull;
+                    ];
+.
+
+ids:AppUpdateResponseMessage a owl:Class;
+    rdfs:subClassOf ids:ResponseMessage ;
+    rdfs:label "App Update Response Message"@en ;
+    rdfs:comment "Message that follows up an AppUpdateRequestMessage and contains the app Update confimation."@en ;
+.
+
+
+ids:AppNotificationMessage a owl:Class;
+    rdfs:subClassOf ids:ResourceNotificationMessage ;
+    idsm:abstract true ;
+    rdfs:label "App Notification Message"@en ;
+    rdfs:comment "Superclass of all messages, indicating a change of an DataApp."@en;
```

Comment:
"Superclass of all app notification messages, indicating a change of a DataApp."

Reply:
Done :)